spec/components/post_action_creator_spec.rb

```diff
@@ -11,6 +11,10 @@
       RateLimiter.enable
     end
 
+    after do
+      RateLimiter.disable
```

Comment:
Will it be better to move the disabling of rate limiting into an after(:each) instead? https://github.com/discourse/discourse/blob/646cdfa4493d7809c0afdc19adc23ba619c3c789/spec/rails_helper.rb#L165

Reply:
@tgxworld, I completely agree. The next commits actually make this unnecessary so it's unlikely that this one will make it into the final PR.